src/Shell/AclShell.php

```diff
@@ -367,7 +367,19 @@ public function view()
      */
     public function initdb()
     {
-        return $this->dispatchShell('schema create DbAcl');
+        $_SERVER['argv'][1]='migrations';
+        $_SERVER['argv'][2]='migrate';
+        $_SERVER['argv'][3]='--plugin=acl';
```

Comment:
Ok, but when I tried to dispatchShell without change argv I was getting an Exception:

 [InvalidArgumentException]        
  Command "initdb" is not defined.